package.json

```diff
@@ -28,6 +29,11 @@
   },
   "homepage": "https://github.com/NicholasBoll/cypress-pipe#readme",
   "devDependencies": {
-    "cypress": "^2.1.0"
+    "cypress": "^3.1.0",
+    "http-server": "^0.11.1"
+  },
+  "scripts": {
+    "start": "http-server . -p 3333",
```

Comment:
what is this for?

Reply:
That starts a web server for the tests on this project